mobile/native/src/config/api.rs

```diff
@@ -18,8 +18,12 @@ pub struct Config {
     pub health_check_interval_secs: u64,
 }
 
-impl From<Config> for ConfigInternal {
-    fn from(config: Config) -> Self {
+impl From<(Config, String, String)> for ConfigInternal {
```

Comment:
I didn't see where this is being used but this from implementation looks like it's easy to confuse `data_dir` and `seed_dir`.

Reply:
419342e73ffb7ce7338a7c94ca600dc52f3411b3